src/Apps/Settings/Routes/SavedSearchAlerts/SavedSearchAlertsApp.tsx

```diff
@@ -138,12 +138,33 @@ export const SavedSearchAlertsApp: React.FC<SavedSearchAlertsAppProps> = ({
       saved_search_id: editAlertEntity?.id,
     })
 
-    closeEditFormAndRefetch()
+    isMobile ? refresh() : closeEditFormAndRefetch()
     closeDeleteModal()
 
     sendToast({
       message: "Your Alert has been deleted.",
     })
+
+    if (isMobile) return
+
+    if (editAlertEntity?.id === alerts[0].internalID) {
+      setEditAlertEntity({
+        id: alerts[1].internalID,
```

Comment:
are we always sure we have `alerts[1]`, wondering about the case where we have only one alert

Reply:
How did i miss that 🥴  
Corrected the condition, thank you